### ISSUE TITLE
How can I map ctrl+; to a command?

### ISSUE THREAD

**Issue by Inori** [jaredpar/VsVim#1824]
Things like `imap <C-;> <ESC>$a;` doesn't work.

**Comment by jaredpar** [comment]
What command are you trying to map to here?  I'm not sure what `$a` is intended to do.

**Comment by rjindel** [comment]
@OP, you can use  A (as in capital a), to go to the end of the line and enter append mode.
@jaredpar I think the OP wants to map <C-;> to exit insert mode, go the end of the line, and append a ;
I tend to do this quite often and was considering what would be best to map this to <C-;> makes sense if it works

**Comment by op** [comment]
@rjindel Yes, I know. You can use O to begin a new line above the cursor. :)

**Comment by rjindel** [comment]
haha, I meant the Original Poster,

**Comment by sqlkoala** [comment]
The problem seems to be the <C-;> part. I did not get that to work in vsvim and vim either.
However, this works (given that you let vsvim handle Ctrl+e):
inoremap <C-e> <End>;
